gui/packages/ubuntupro/lib/pages/subscribe_now/subscribe_now_widgets.dart

```diff
@@ -1,25 +1,27 @@
-import 'package:dart_either/dart_either.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:provider/provider.dart';
+import 'package:url_launcher/url_launcher_string.dart';
 import 'package:yaru/yaru.dart';
-import '../../core/either_value_notifier.dart';
 import '../../core/pro_token.dart';
+import 'subscribe_now_model.dart';
 
 /// A validated text field with a submit button that calls the supplied [onApply]
 /// callback with the validated Pro Token when the submit button is clicked.
 class ProTokenInputField extends StatefulWidget {
```

Comment:
Since the text editing controller is managed elsewhere I no longer see why this has to be a stateful widget.